app/home/controllers.js

```diff
@@ -3,15 +3,19 @@ const { get } = require('lodash')
 const dateHelpers = require('../../common/helpers/date')
 const permissions = require('../../common/middleware/permissions')
 
-function dashboard(req, res) {
+function movesRedirect(req, res, next) {
```

Comment:
Should this live in `middleware` instead of `controllers`? Feels like it resembles what we've called middleware elsewhere in the app.

Reply:
Lazily just extracted the method to the same module... you're right